content/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -0,0 +1,49 @@
+---
+title: Tutorial Interaktif - Men-<i>Deploy</i> Aplikasi
```

Comment:
Italics in the title will not work (it is not yet HTML markup). Also, please give some Indonesian words.
```suggestion
title: Tutorial Interaktif - Menyebarkan (Deploy) Aplikasi
```

Reply:
how about menjalankan?
```suggestion
title: Tutorial Interaktif - Menjalankan Aplikasi
```